### ISSUE TITLE
Focus outline is missing on sort column buttons

### ISSUE THREAD

**Issue by rilrom** [payloadcms/payload#9611]
### Describe the Bug

When using the keyboard, focus is invisible when the active element is a sort column button.

Take a look at this [PR](https://github.com/payloadcms/payload/pull/9557) for more information.

### Link to the code that reproduces this issue

https://github.com/payloadcms/payload/tree/main/test/_community

### Reproduction Steps

1. Navigate to the list view.
2. Using the keyboard, tab to the sort column buttons.
3. No focus outline is shown.

### Which area(s) are affected? (Select all that apply)

area: ui

### Environment Info

```text
Payload: 3.2.1
Next.js: 15.0.3
```
